server/.gitignore

```
@@ -16,6 +16,8 @@ schema.graphql
 *.sqlite-journal
 *.sqlite-wal
 *.sqlite-shm
+# test template DB marker
+___template_needs_update.marker
```

the markers are prefixed with the template dbs, so would need a wildcard to start:
```suggestion
*___template_needs_update.marker
```

I expected that a glob format is required (`**/*___template_needs_update.marker`) but seems fine without that 🤷

---

Not quite sure while * is needed? the marker file name is quite unique?

Just had a look at the docs: https://git-scm.com/docs/gitignore and this should be the rule that is used here:

`If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.`